AutoRunCustomTool/Options/OptionsPageGrid.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Microsoft.VisualStudio.Shell;
+using System.ComponentModel;
+
+namespace ThomasLevesque.AutoRunCustomTool.Options
+{
+    public class OptionsPageGrid : DialogPage
+    {
+        [Category("AutoRunCustomTool")]
+        [DisplayName("Extensions")]
+        [Description("What extension(s) to run the custom tool on")]
+        public string ListenToExtension { get; set; }
```

Comment:
The name should be plural, since it can contain multiple extensions.

Also, I'd rename it to something clearer. Maybe `TriggerExtensions`?

Reply:
Renamed the DisplayName, Description and variable name to better reflect what the purpose of it is.